typings/index.d.ts

```diff
@@ -5971,6 +5971,7 @@ declare namespace KubernetesClient {
     'version': Version
     // Calls
     loadSpec(): Promise<Api>
+    addCustomResourceDefinition(schema: object): void;
```

Comment:
index.d.ts is machine generated.  Can you add this to the template (https://github.com/godaddy/kubernetes-client/blob/master/scripts/templates/ts-interface.mustache#L14)?